[2023-03-01 13:00:04,210] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-11-26T23:05:00+00:00 [queued]>
[2023-03-01 13:00:04,318] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-11-26T23:05:00+00:00 [queued]>
[2023-03-01 13:00:04,320] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 13:00:04,320] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2023-03-01 13:00:04,321] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 13:00:04,486] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): get_name> on 2022-11-26T23:05:00+00:00
[2023-03-01 13:00:04,528] {standard_task_runner.py:52} INFO - Started process 17069 to run task
[2023-03-01 13:00:04,648] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'with_python_operator', 'get_name', '2022-11-26T23:05:00+00:00', '--job-id', '4058', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/02_python_operator.py', '--cfg-path', '/tmp/tmpd10pvfsl', '--error-file', '/tmp/tmpesrlo1hb']
[2023-03-01 13:00:04,649] {standard_task_runner.py:77} INFO - Job 4058: Subtask get_name
[2023-03-01 13:00:05,154] {logging_mixin.py:104} INFO - Running <TaskInstance: with_python_operator.get_name 2022-11-26T23:05:00+00:00 [running]> on host abe4c4997ef7
[2023-03-01 13:00:05,705] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dataheroes
AIRFLOW_CTX_DAG_ID=with_python_operator
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2022-11-26T23:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-26T23:05:00+00:00
[2023-03-01 13:00:05,869] {python.py:118} INFO - Done. Returned value was: None
[2023-03-01 13:00:06,011] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=with_python_operator, task_id=get_name, execution_date=20221126T230500, start_date=20230301T130004, end_date=20230301T130006
[2023-03-01 13:00:06,395] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-03-01 13:00:06,513] {local_task_job.py:146} INFO - Task exited with return code 0
